hubble-fe/src/components/graph-management/data-analyze/DataAnalyzeInfoDrawer.tsx

```diff
@@ -207,7 +201,11 @@ const DataAnalyzeInfoDrawer: React.FC = observer(() => {
           ([key, value], primaryKeyIndex) => (
             <div className={graphInfoItemClassName} key={key}>
               <div>
-                {key}(主键{`${primaryKeyIndex + 1}`})：
+                {key}(
+                {dataAnalyzeStore.graphInfoDataSet === 'node'
+                  ? '主键'
+                  : '区分键'}
+                {`${primaryKeyIndex + 1}`})：
```

Comment:
add a function

Reply:
For now i18n does not applied in v1.0 - v1.2 (added in > v1.3.0 in comparison), which needs some extra works since v1.2.2 changes lots of code in repo. If i18n applies there's no hardcoded text any more.